packages/http/src/mocker/__tests__/HttpMocker.spec.ts

```diff
@@ -226,7 +226,12 @@ describe('HttpMocker', () => {
             const response = httpMocker.mock({
               input: mockInput,
               resource: mockResource,
-              config: { cors: false, mock: { dynamic: false, exampleKey: 'test key' } },
+              config: {
+                cors: false,
+                mock: { dynamic: false, exampleKey: 'test key' },
+                validateRequest: true,
+                validateResponse: true,
```

Comment:
Is it required to pass this in? We have it in all the tests, and that along with cors makes this all look rather tricky.

Reply:
I can probably DRY this up and reuse the object but yes they are mandatory on purpose. See the comment below for an explanation.